xtext-website/documentation/340_lsp_support.md

```diff
@@ -73,9 +73,37 @@ Xtext supports the generation of language servers that comply to the [Language S
 }
 ```
  1. Open a `*.dmodel` file with the Generic Editor and explore the supported language features like syntax highlighting, content assist, validation, displaying code lenses, quickfixes, formatting, ... :
```

Comment:
unfortunately this is currently broken see https://github.com/eclipse/lsp4e/issues/239